packages/discord-bot-ui/src/app/common/signIn/signIn.ts

```diff
@@ -4,7 +4,7 @@
  * SPDX-License-Identifier: MIT
  */
 import { DAppClient, NetworkType, RequestSignPayloadInput } from '@airgap/beacon-sdk'
-import { useSiwt } from '@siwt/react'
```

Comment:
Is the react package now deprecated and are you deleting it?

Reply:
Not officially, but it should be.